config/horizon.php

```diff
@@ -145,59 +145,83 @@
             'supervisor-bpmn' => [
                 'connection' => 'redis',
                 'queue' => ['bpmn'],
-                'balance' => 'simple',
-                'processes' => 16,
-                'tries' => 1,
-                'timeout' => 3600,
+                'balance' => env('PM4_HORIZON_SUPERVISOR_BPMN_BALANCE', 'simple'),
+                'processes' => env('PM4_HORIZON_SUPERVISOR_BPMN_PROCESSES', 4),
```

Comment:
Reviewing the documentation, looks like 'processes' are not compatible with 'minProcesses' and 'maxProcesses', we need to pick one only.

Reply:
@epdl I think this is based on the `balance` setting. If balance is simple or false, the `processes` setting is used. If balance is `auto`, then `processes` is ignored and the `minProcesses` and `maxProcesses` is used.